Cargo.toml

```diff
@@ -59,6 +59,8 @@ testcontainers = "0.16.7"
 testcontainers-modules = { version = "0.4.2", features = ["localstack"] }
 time = { version = "0.3.34", features = ["serde"] }
 geojson = "0.24.1"
+rand = { version = "0.8.5" }
+approx = "0.5.1"
```

Comment:
seem unused

Reply:
Both crates are used in the module path `tests/fixtures/tables/auto_sales.rs`.